.env

```diff
@@ -0,0 +1,28 @@
+# Development URLS
+
+REACT_APP_REALM = ClientFhirServer
+REACT_APP_CLIENT = app-login
+REACT_APP_AUTH = http://localhost:8180
+REACT_APP_SERVER = http://localhost:8090
```

Comment:
Unsure where its used or if it is but since it was defined before I just changed it over to keep it just in case so we don't break it right before a release. I think in the future we can have a ticket to go through and remove unused things after proper testing since its a little out of scope of the ticket.